rest/session_api.go

```diff
@@ -254,7 +255,7 @@ func (h *handler) deleteUserSession() error {
 // ADMIN API: Deletes all sessions for a user
 func (h *handler) deleteUserSessions() error {
 	h.assertAdminOnly()
-
+	h.setHeader("Content-Type", "application/json")
```

Comment:
And here - no response body, no content-type.

Reply:
Yeah; this makes more sense. Fixed in latest revision.